examples/fake-ad.amp.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
```

Comment:
Do customers know to look there? What is the intended difference between an example and a manual test?